<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="arun" failures="1" tests="1" name="Test" time="9.07" errors="0" timestamp="2024-07-17T10:59:01 IST">
  <testcase classname="testScript.LoginTest" name="verifyTheUserIsAbleToLoginUsingValidCredentials" time="3.69">
    <failure type="java.lang.AssertionError" message="Page is not loaded when user gives valid credentials expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Page is not loaded when user gives valid credentials expected [false] but found [true]
at testScript.LoginTest.verifyTheUserIsAbleToLoginUsingValidCredentials(LoginTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyTheUserIsAbleToLoginUsingValidCredentials -->
  <testcase classname="testScript.LoginTest" name="verifyTheUserIsAbleToLoginUsingValidUsernameAndInvalidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTheUserIsAbleToLoginUsingValidUsernameAndInvalidPassword -->
  <testcase classname="testScript.LoginTest" name="verifyTheUserIsAbleToLoginUsingInvalidUsernameAndValidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTheUserIsAbleToLoginUsingInvalidUsernameAndValidPassword -->
  <testcase classname="testScript.LoginTest" name="verifyTheUserIsAbleToLoginUsingInvalidUsernameAndInvalidPassword" time="0.0">
    <ignored/>
  </testcase> <!-- verifyTheUserIsAbleToLoginUsingInvalidUsernameAndInvalidPassword -->
</testsuite> <!-- Test -->
